main.css

```diff
@@ -0,0 +1,1029 @@
+    .navbar {
+    display: flex;
+    align-items: center;
+    justify-items: center;
+    justify-content: space-between;
+    color: var(--primary-color);
+    padding: 1rem ;
+    
+    }
+    
+    .active {
+    font-weight: bold;
+    }
+    
+    .navbar ul{
+      margin: 0;
+      padding: 0;
+    }
+    
+    .navbar li {
+    display: inline;
+    margin: 0 1rem;
+    font-size: larger;
+    }
+    
+    .navbar a{
+      text-decoration: none;
+    }
+    
+    .nav-brand{
+    font-size:xx-large;
+    font-weight: 600;
+    }
+    
+    /* nav *********************************************************/
+    
+    
+    /* images **************************************************/
+    
+    .img-responsive {
+    max-width: 100%;
+    height: auto;
+    }
+    .img-round {
+    border-radius: 50%;
+    max-width: 100%;
+    height: auto;
+    }
+    
+    /* images **************************************************/
+    
+    
+    /* forms **********************************************/
+    
+    [type=color], [type=date], [type=datetime], [type=datetime-local], [type=email], [type=month], [type=number], [type=password], [type=search], [type=tel], [type=text], [type=url], [type=week], [type=time], select, textarea {
+    display: block;
+    border: 1px solid var(--light-border);
+    border-radius: 0.25rem;
+    padding: 0.75rem;
+    outline: none;
+    background: transparent;
+    margin-bottom: 0.5rem;
+    font-size: 1rem;
+    width: 100%;
+    max-width: 100%;
+    line-height: 1; }
+    
+    [type=color]:hover, [type=date]:hover, [type=datetime]:hover, [type=datetime-local]:hover, [type=email]:hover, [type=month]:hover, [type=number]:hover, [type=password]:hover, [type=search]:hover, [type=tel]:hover, [type=text]:hover, [type=url]:hover, [type=week]:hover, [type=time]:hover, select:hover, textarea:hover {
+    border: 1px solid var(--less-light-border); }
+    
+    [type=color]:focus, [type=color]:active, [type=date]:focus, [type=date]:active, [type=datetime]:focus, [type=datetime]:active, [type=datetime-local]:focus, [type=datetime-local]:active, [type=email]:focus, [type=email]:active, [type=month]:focus, [type=month]:active, [type=number]:focus, [type=number]:active, [type=password]:focus, [type=password]:active, [type=search]:focus, [type=search]:active, [type=tel]:focus, [type=tel]:active, [type=text]:focus, [type=text]:active, [type=url]:focus, [type=url]:active, [type=week]:focus, [type=week]:active, [type=time]:focus, [type=time]:active, select:focus, select:active, textarea:focus, textarea:active {
+    border: 1px solid var(--primary-blue);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.1), 0 0 6px #8cbcfe; }
+    
+    textarea {
+    overflow: auto;
+    height: auto; }
+    
+    fieldset {
+    border: 1px solid #dedede;
+    border-radius: 4px;
+    padding: 1rem;
+    margin: 1.5rem 0; }
+    
+    legend {
+    padding: 0 0.5rem;
+    font-weight: 600; }
+    
+    select {
+    color: #404040;
```

Comment:
Hi Rakesh,
You should use a css variable here too.

Reply:
noted bro